sdk/synapse/Azure.Analytics.Synapse.AccessControl/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-preview.3 (Unreleased)
+## 1.0.0-beta.3 (Unreleased)
```

Comment:
Do we need to change this to `2020-11-12` just like Azure.Analytics.Synapse.Artifacts/CHANGELOG.md

Reply:
No we don't release it this time.